.travis.yml

```diff
@@ -34,13 +34,16 @@ before_cache:
 
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
What was the issue you saw with oraclejdk8?

Reply:
the issue is simar to: https://travis-ci.community/t/expected-feature-release-number-in-range-of-9-to-12-but-got-8-installing-oraclejdk8

Reply:
Thanks